src/database/migrations/20201013100651-update-accesstoken.js

```diff
@@ -1,12 +1,12 @@
 module.exports = {
   up: async (queryInterface, Sequelize) => {
     await queryInterface.changeColumn('AccessTokens', 'token', {
-      type: Sequelize.STRING(1024)
+      type: Sequelize.STRING(1234)
     });
   },
   down: async (queryInterface, Sequelize) => {
-    await queryInterface.changeColumn('AccessTokens', 'token', {
-      type: Sequelize.STRING(255)
+    await queryInterface.removeColumn('AccessTokens', 'token', {
+      type: Sequelize.STRING
     });
```

Comment:
Can you keep the previous changes here?

Reply:
I think when we run build and npm start then migration undo all is not running because of this migration, I am going to find a way to handle this no prob

Reply:
solved